src/errors/test-run/index.js

```diff
@@ -158,6 +158,26 @@ export class ActionPositiveIntegerArgumentError extends ActionArgumentErrorBase
     }
 }
 
+export class ActionStringOrStringArrayArgumentError extends ActionArgumentErrorBase {
+    constructor (argumentName, actualValue) {
+        super(TYPE.actionStringOrStringArrayArgumentError, argumentName, actualValue);
+
+        if (Array.isArray(actualValue) && !actualValue.length)
+            this.actualValue = '[]';
+        else if (actualValue === '')
+            this.actualValue = '""';
```

Comment:
Got it. But why we have part of the logic in validator and here? Can we just always pass `actualValue` and assign `[]` or `""` in validator?